package.json

```diff
@@ -24,21 +22,21 @@
     "test:coverage": "npm run lint && npx vitest run --coverage"
   },
   "peerDependencies": {
-    "@mui/material": "^6.x",
-    "@mui/system": "^6.x",
+    "@mui/material": "^7.x",
+    "@mui/system": "^7.x",
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
     "react-i18next": "^13.0.0 || ^14.0.0 || ^15.0.0",
     "lodash": "^4.17.11",
-    "mirador": "^4.0.0-alpha.15",
+    "mirador": "^4.0.0-alpha.17",
     "react": "^18.x || ^19.x",
     "react-dom": "^18. || ^19.x"
```

Comment:
`^18. ` -- this isn't a change you made, but it is causing me an issue with testing in an application as it is a syntax error. We should fix.

Reply:
Thanks. Just to be clear the issue was a period with nothing following -- `^18.` -- not the other syntax, but it is an improvement!